File: workspace/python/sum.py
def main():
  print('The sum of the list is {}'.format(num_list([1,2,3,4,5])))

def num_list(array):
  total = 0

  for num in array:
    total += num
  
  return total


if __name__ == '__main__':
  main()

File: workspace/java/sum.java
public class sum {
    
    public static void main(String[] argv) {
        int num = 100, sum = 0;

        for(int i = 1; i <= num; ++i)
        {
            // sum = sum + i;
            sum += i;
        }

        System.out.println("Sum = " + sum);;
    }
}

